### ISSUE TITLE
[iceprod2] global state is evil (Trac #1292)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1286]
`iceprod.core.exe.config` is global state.  Find a way to make this better.

This breaks the test suite when testing `core.i3exec` and `core.exe` in the same run.

Migrated from https://code.icecube.wisc.edu/ticket/1292
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "`iceprod.core.exe.config` is global state.  Find a way to make this better.\n\nThis breaks the test suite when testing `core.i3exec` and `core.exe` in the same run.", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] global state is evil", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-25T22:38:23", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150828T22:40:27: 
* david.schultz commented:

> In changeset 136989:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136989"
> get rid of config global state. fixes #1286
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
